docs/content/en/install/eks/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Install K8ssandra on EKS"
+linkTitle: "Amazon EKS"
+weight: 2
+description: >
+  Complete **production** ready environment of K8ssandra on Amazon Elastic Kubernetes Service (EKS).
+---
+
+Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/features/) or "EKS" is a managed Kubernetes service that makes it easy for you to run Kubernetes on AWS and on-premises. EKS is certified Kubernetes conformant, so existing applications that run on upstream Kubernetes are compatible with EKS. AWS automatically manages the availability and scalability of the Kubernetes control plane nodes responsible scheduling containers, managing the availability of applications, storing cluster data, and other key tasks.
+
+{{% alert title="Tip" color="success" %}}
+Also available in followup topics are post-install steps and role-based considerations for [developers]({{< relref "/quickstarts/developer">}}) or [site reliability engineers]({{< relref "/quickstarts/site-reliability-engineer">}}) (SREs).
+{{% /alert %}}
+
+## Requirements
+
+1. Amazon EKS Cluster - Follow the [Amazon EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html)
+   a. K8ssandra has been tested with `Managed nodes - Linux` instances. `Fargate - Linux` _may_ work, but has not been validated or tested. If you are interested in support for Fargate, please [open an issue](https://github.com/k8ssandra/k8ssandra-operator/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=Support+for+Fargate+on+EKS).
+   b. IAM Role attached to EKS EC2 instances _must_ include permissions to Amazon S3 APIs if you are going to be leveraging backup and restore functionality. See this [IAM policy](https://github.com/k8ssandra/k8ssandra-terraform/blob/ca5b1038d8cee003c0732d5c93ae06592e0a8535/aws/scripts/policy_document.json#L159-L190) as an example. _Note your policy should limit requests to specific buckets and operations._
+2. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to access the cluster
+3. _Optional_ [Helm](https://helm.sh/docs/intro/install/) installed and configured to access the cluster
+4. _Optional_ [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) installed and configured to access the cluster
```

Comment:
```suggestion
1. Amazon EKS Cluster - Follow the [Amazon EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html).
   a. K8ssandra is tested with `Managed nodes - Linux` instances. `Fargate - Linux` _may_ work, but has not been validated or tested. If you are interested in support for Fargate, please [open an issue](https://github.com/k8ssandra/k8ssandra-operator/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=Support+for+Fargate+on+EKS).
   b. IAM Role attached to EKS EC2 instances _must_ include permissions to Amazon S3 APIs if you are going to be leveraging backup and restore functionality. See this [IAM policy](https://github.com/k8ssandra/k8ssandra-terraform/blob/ca5b1038d8cee003c0732d5c93ae06592e0a8535/aws/scripts/policy_document.json#L159-L190) as an example. _Note your policy should limit requests to specific buckets and operations._
2. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to access the cluster.
3. _Optional_ [Helm](https://helm.sh/docs/intro/install/) installed and configured to access the cluster.
4. _Optional_ [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) installed and configured to access the cluster.
```
If you are specifically talking about DataStax not validating or testing Fargate - Linux, you might want to say that.

Reply:
I'm alright with this wording regarding Fargate.